applicationset/services/scm_provider/gitlab_test.go

```diff
@@ -79,11 +79,38 @@ func TestGitlabHasPath(t *testing.T) {
 		Repository:   "argocd",
 		Branch:       "master",
 	}
-	ok, err := host.RepoHasPath(context.Background(), repo, "argocd")
-	assert.Nil(t, err)
-	assert.True(t, ok)
 
-	ok, err = host.RepoHasPath(context.Background(), repo, "notathing")
-	assert.Nil(t, err)
-	assert.False(t, ok)
+	cases := []struct {
+		name, path string
+		exists     bool
+	}{
+		{
+			name:   "directory exists",
+			path:   "argocd",
+			exists: true,
+		},
```

Comment:
Would `argocd/` have worked before / does it still work?

Reply:
Yes would have worked doesn't matter